gbpservice/nfp/orchestrator/config_drivers/heat_driver.py

```diff
@@ -246,6 +246,20 @@ def loadbalancer_post_stack_create(self, network_function_details):
                     if port_info['port_model'] != nfp_constants.GBP_PORT:
                         return
 
+    def _post_stack_create(self, nfp_context):
+        service_details = self.get_service_details_from_nfp_context(
+            nfp_context)
+        service_details = service_details['service_details']
+        service_type = service_details['service_type']
+
+        if service_type in [pconst.LOADBALANCER]:
+            logging_context = nfp_logging.get_logging_context()
+            auth_token = logging_context['auth_token']
```

Comment:
This auth_token is coming from gbp for each api,during the execution of nfp.we are storing it in execution thread.and max time for execution of any request from gbp is 10 minutes and expiry_time for auth_token is 30 minutes,so it should never expired during our execution.